### ISSUE TITLE
Double-click To Give An Item Bug?

### ISSUE THREAD

**Issue by Rhett008** [ricksouth/serilum-mc-mods#2229]
## **Information**
Minecraft version: 1.20.1
Modloader: Fabric
Fabric loader version: 0.15.6
Environment: Multiplayer

Mod name: Hand Over Your Items
Mod version: 3.2

[mod config is default]

## **Description**
Hi, I was testing this mod on a new fabric server. Attempting to give an item in hand by crouch-clicking another player will result in the item disappearing from the giver's hand, and no give/receive message in the chat will appear (as if the item is lost in a "transfer void"). Only until you crouch-click the player again with the - now empty - hand, the item is transferred to the receiving player and both parties get a give/receive message in the chat respectively.

Basically, transferring an item or item stack requires crouch-clicking a player twice instead of once to properly work (I assume that a transfer is intended to work with only 1 click and not 2). This is a little frustrating when;
the item can be lost in a "void" somewhere and may not be retrievable if a double-click doesn't occur within a certain timeframe,
the giver switches hotbar spaces before the transfer is complete (mid-transfer), or
either player dies mid-transfer.

I'm not sure if this is a bug or an intended feature, but I couldn't find any mention of it elsewhere so I am willing to bet it is unintentional.

**Comment by ricksouth** [comment]
Indeed sounds like a bug, thanks for opening the issue. Was the mod also installed on the client, or server-side only?

**Comment by Rhett008** [comment]
> Indeed sounds like a bug, thanks for opening the issue. Was the mod also installed on the client, or server-side only?

The mod was installed on the server and both player clients, including the Collective Lib

**Comment by Rhett008** [comment]
@ricksouth is there any update?

**Comment by guilhermufc** [comment]
same here
